### PR TITLE
GEN-1339: add eslint to deploments pkg [QUALITY WORK]

### PR DESCRIPTION
**Pull request by fedealconada**
## Motivation

Improve code quality

## Solution

Improve code quality by adding eslint + prettier to the `/deployments` packcage.
It adds 2 new scripts:
`yarn lint` -> only lints code
`yarn fix` -> lints and tries fixing what's possible

## Implementer Checklist

- [ ]  [FIRST TIME ONLY] Review the [Solcurity](https://github.com/Rari-Capital/solcurity) standard
- [ ]  [OPTIONAL] Create a reference implementation (in python or JS)
- [x]  Create a short "Motivation" section for the PR
- [x]  Write a spec for the feature and put it in the PR description – basic function names and expected state transitions is OK
- [ ]  Check all of the new revert paths with concrete tests
- [ ]  Check all of the new storage slot writes with concrete tests – do not make the passing test case the trivial case!
- [ ]  Go line-by-line through the new code and ensure it has all been covered by concrete tests – since we don’t have a coverage engine for foundry yet, you can imagine how one might check that each different code path is covered
- [ ]  Simplify the implementation and spend some time trying to minimize gas costs
- [ ]  Write fuzz tests for the feature – try everything to break the implementation (is this function monotonically in/decreasing, should it always be less than something else, etc)
- [ ]  Capture bugs discovered during the above steps in concrete tests (regression tests)
- [ ]  Add integration tests – fuzz or concrete tests that test how the new code affects the entire system, either locally or with a mainnet fork
- [ ]  [OPTIONAL] Integrate the feature into the deployment scripts
- [ ]  [OPTIONAL] Add new sanity checks to the deployment scripts
- [ ]  Get the PR reviewed by at least two people

### CODE REVIEW

File: pkg/deployments/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "semi": true,
```
Comment (jparklev) [inline pkg/deployments/.prettierrc:2]: What're your thoughts on just taking the defaults for all of these? (except for `printWidth` which I agree with)

**Review by jparklev — APPROVED** [review]
so pretty

**Comment by abylaw** [comment]
p r e t t y

we could also lint on commit ~ https://typicode.github.io/husky/#/
for example: https://github.com/sense-finance/web3-yield-curves

File: pkg/deployments/.prettierrc (hunk 1 above)
Comment (fedealconada) [reply pkg/deployments/.prettierrc:2]: let's do it!

**Comment by fedealconada** [comment]
> p r e t t y
> 
> we could also lint on commit ~ https://typicode.github.io/husky/#/ for example: https://github.com/sense-finance/web3-yield-curves

added!

**Review by abylaw — APPROVED** [review]

**Pull request merged**